_docs/tutorial/99-cleanup.md

```diff
@@ -25,5 +25,5 @@ type: markdown
 
 ## What's next
 {% if page.previous.url %}
-Previous step: [{{page.previous.title}}]({{page.previous.url}})
+Previous step: [{{page.previous.title}}]({{home}}/{{page.previous.url}})
```

Comment:
@geeknoid it seems like a bug in "page.previous.url" if it needs this prefix, no ?

Reply:
@ldemailly @geeknoid `page.previous.url` and `page.next.url` are the Jekyll constructs. I saw it in http://david.elbe.me/jekyll/2015/06/20/how-to-link-to-next-and-previous-post-with-jekyll.html.

Adding `{{home}}` (without a slash) to `page.previous.url` and `page.next.url` works and the rake test is happy.